packages/atomic/src/global.pcss

```diff
@@ -6,5 +6,19 @@ atomic-search-interface:not(.hydrated) {
 }
 
 :root {
```

Comment:
I took this pallet directly from React Vapor including the input height but as I mentioned they have a ton of variables so we probably can't just do a 1 for 1 copy. They defined these colors as the "primary pallet" so it seemed like the logical choice to use.

Reply:
For some of these, we must be careful since they don't all have a very high contrast ratio against white:
* dark-medium-grey: 4.13:1
* medium-grey: 1.78:1
* light-grey: 1.19:1
* orange: 2.62:1

Reply:
I'm gonna make the greys darker, the orange is actually the "Coveo" orange so I can't change that.

Reply:
These colour values can be used for the default Coveo look. However, if we want Atomic to be theme-able, we should not name our variables e.g. "darker-blue", but rather use more abstract names. Things like:

primary, secondary, accent, warning, error and so on.

In the Coveo theme, the primary variable might then we assigned value `#263e55` (aka dark-blue).

Choosing the right variables is important. I would look at what other frameworks/design systems typically offer, since Atomic users will probably expect at least a similar degree of freedom.

Bootstrap example: https://getbootstrap.com/docs/4.0/utilities/colors/

Reply:
Another example is lightning design system token:

https://www.lightningdesignsystem.com/design-tokens/#category-color

But, as Sami said, these need to be "generic".

However, don't go overboard with overcomplicating this *for right now*, because this will most probably change when UX gets on board.... so.

My suggestion for now: 

* primary color
* secondary color
* tertiary color
* text with light background
* text with dark background
* font family
* font size small / medium / large
* padding small / medium / large
* margin small / medium / large
* border (radius/width)


If we quickly feel that we are not able to get something "meaningful", or we are missing something, then we can add them as we go.

It's just a suggestion, and I'm sure some things are missing...

Reply:
I agree but we do need to be careful that its intuitive for end users to understand what each of the colors is being used for. Font family/font size also makes sense to me. I'm personally not in favour of exposing padding and margin variables, they'll affect too many places and probably not be all that useful in practice. I think over-writing the margins/padding using parts on specific components makes more sense.

Reply:
I was quickly looking into design systems for colors and the [Material UI coloring scheme](https://material.io/design/color/the-color-system.html#color-theme-creation) seems to be intuitive to me.

I much prefer this to bootstraps system because the "On Primary", "On Secondary" tags for colors make more sense to me than Bootstraps "text-primary", "bg-primary" because for the Searchbox the border and the text should probably be the same color because they're both on the same background. 

With a bootstrap style system the user would have to change "text-primary" and "bg-primary" to get both to change. Also having "bg-primary" as blue might make sense but having "text-primary" as blue probably doesn't. But if the colors for "bg-primary" and "text-primary" don't match it makes it confusing to change. 

With the material theme however, since neither the text nor the border are "accent colors", we would just use the "On Background" color, so both the text and the background would be gray. Obviously we can add to the theme/remove from it where it makes sense but I think the approach of thinking of colors based upon layers and how they relate to each other is much better than the bootstrap system.

Reply:
Ok ! 

If you had to define a list of variable that we'd use, what would it look like ?

Reply:
primary (probably coveo blue)
on-primary
secondary (probably coveo orange)
on-secondary
background
on-background (a gray that contrasts well with white)
error (some red)
on-error

Reply:
Sounds good! I would still prepend those variables with "atomic-" thought.
Instead of the orange, we should use one of the accent color, and/or secondary colors from the new branding https://brand.coveo.com/d/cquweveUA42v/visual-identity#/style-guide/colors